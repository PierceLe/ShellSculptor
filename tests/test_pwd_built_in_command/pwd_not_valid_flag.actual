>> cd ~/test_data/PWD_TEST
cd: no such file or directory: /Users/piercele/test_data/PWD_TEST
>> pwd -fP
pwd: invalid option: -f
>> pwd -Pm
pwd: invalid option: -m
>> pwd -t -P
pwd: invalid option: -t
>> pwd -abcdP
pwd: invalid option: -a
>> pwd hi
pwd: not expecting any arguments
>> pwd -P -P -P
/Users/piercele/Desktop/year-2/sem1/info1112/ass1/ShellSculptor/tests
>> pwd -P -t
/Users/piercele/Desktop/year-2/sem1/info1112/ass1/ShellSculptor/tests

