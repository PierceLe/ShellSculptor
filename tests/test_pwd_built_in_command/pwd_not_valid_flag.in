cd ~/test_data/PWD_TEST
pwd -fP
pwd -Pm
pwd -t -P
pwd -abcdP
pwd hi
pwd -P -P -P
