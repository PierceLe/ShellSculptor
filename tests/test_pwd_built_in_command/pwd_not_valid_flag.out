>> cd ~/test_data/PWD_TEST
>> pwd -fP
pwd: invalid option: -f
>> pwd -Pm
pwd: invalid option: -m
>> pwd -t -P
pwd: invalid option: -t
>> pwd -abcdP
pwd: invalid option: -a
>> pwd hi
pwd: not expecting any arguments
>> pwd -P -P -P
/home/test_data/PWD_TEST

